host/3.0/nanoserver/1809/dotnet.Dockerfile

```diff
@@ -6,7 +6,7 @@ FROM mcr.microsoft.com/windows/servercore:ltsc2019 AS installer-env
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 # Retrieve .NET Core SDK
-RUN $dotnet_sdk_version = '3.1.408'; `
+RUN $dotnet_sdk_version = '3.1.412'; `
```

Comment:
instead of hardcoding a dotnet sdk version - is it possible to use the latest 3.1.x?

Reply:
I will investigate this now. I agree that seems like a good idea :)

Reply:
That is how the dotnet team does it. This part is copied (and should be kept in sync with) this file https://github.com/dotnet/dotnet-docker/blob/main/src/sdk/3.1/nanoserver-1809/amd64/Dockerfile#L12

Reply:
I see. They manually update the version. Makes sense. @CooperLink lets go ahead with this change for now. Lets revisit if manually updating every release is a problem in future.